examples/26-install-module.ts

```diff
@@ -0,0 +1,60 @@
+import { BigNumber, ethers } from 'ethers';
+import { EtherspotBundler, PrimeSdk } from '../src';
+import { printOp } from '../src/sdk/common/OperationUtils';
+import * as dotenv from 'dotenv';
+import { MODULE_TYPE, sleep } from '../src/sdk/common';
+
+dotenv.config();
+
+async function main() {
+  const bundlerApiKey = 'eyJvcmciOiI2NTIzZjY5MzUwOTBmNzAwMDFiYjJkZWIiLCJpZCI6IjMxMDZiOGY2NTRhZTRhZTM4MGVjYjJiN2Q2NDMzMjM4IiwiaCI6Im11cm11cjEyOCJ9';
```

Comment:
Is this ok to be left in the test file?

Reply:
This is the default API key, which we have previously included in the example file.